src/mrdsrg-spin-adapted/sadsrg.cc

```diff
@@ -260,7 +265,8 @@ void SADSRG::check_init_memory() {
 
     mem_left -= n_ele * sizeof(double);
     if (mem_left < 0) {
-        throw psi::PSIEXCEPTION("Not enough memory to run FORTE.");
+        std::string msg = "Not enough memory to run spin-adapted DSRG. Try DiskDF integrals?";
```

Comment:
This is a great function. Can we tell the user how much more memory is needed?

Reply:
This is a good point. I have added a few lines for printing.